RecoTauTag/RecoTau/plugins/DPFIsolation.cc

```diff
@@ -168,8 +168,6 @@ class DPFIsolation : public deep_tau::DeepTauBase {
 
         if (p.pt() < 0.5)
           continue;
-        if (p.fromPV() < 0)
-          continue;
```

Comment:
`p.fromPV()` return `PVAssoc` enum which is always `>=0` ( https://cmssdt.cern.ch/lxr/source/DataFormats/PatCandidates/interface/PackedCandidate.h#0703 )

```
RecoTauTag/RecoTau/plugins/DPFIsolation.cc:171:24: error: result of comparison of unsigned enum expression < 0 is always false [-Werror,-Wtautological-unsigned-enum-zero-compare]
```

Reply:
unless the check was supposed to be `p.fromPV() == 0`  (and `p.fromPV() <= 1` below) ?

Reply:
@ocolegrove @MRD2F @mbluj 
It appears that a check "p.fromPV() < 0" is redundant.
please double-check that we are not hiding some problem in your original code.

Reply:
I guess it is fine the remove the `p.fromPV() < 0` redundant check.